lambda/aws-web-crawler-lambda/index.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+ *  with the License. A copy of the License is located at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+ *  OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+ *  and limitations under the License.
+ */
+import 'jquery';
+import { Crawler } from './crawler';
+import { CrawlerRequestTest } from './types';
+
+/*
```

Comment:
please remove this commented urls.

Reply:
Fixed in https://github.com/awslabs/generative-ai-cdk-constructs/pull/474/commits/94869bc149e4ed35dee85ff8a903754189564663